pkg/base1/cockpit.js

```diff
@@ -1280,6 +1280,10 @@ function basic_scope(cockpit, jquery) {
         return deferred.promise;
     };
 
+    cockpit.resolve = function resolve(result) {
```

Comment:
I would say that having the helper functions just take one argument is ok. The expanded code can take more.

In addition resolving or rejecting a promise with multiple arguments, is rather non-standard with regard to the Promises javascript spec. We can have the capability for the cases where it's necessary, but it doesn't need to be routine.